### ISSUE TITLE
Update to Manifest v3

### ISSUE THREAD

**Issue by sam20e** [denilsonsa/crx-reload-tab#14]
Hello,

Firstly, I want to thank you for taking the time to create this extension and make it public.

I'm curious if there are plans to release a new update that aligns with Manifest v3 standards? If so, that would be fantastic.

Thank you.

**Comment by denilsonsa** [comment]
Yes, it is on the pile of stuff to do for my personal side projects… That pile is not small. :(

On the bright side, the V2 deprecation is still postponed with no real timeline yet.
https://developer.chrome.com/docs/extensions/migrating/mv2-sunset/

---

It looks like V3 is dropping support for background pages and replacing them with service workers. While that sounds like a good change, it also means I have to migrate the code and do a lot of testing to make sure it continues working correctly.

**Comment by mattccrampton** [comment]
Thanks again @denilsonsa for working on this whenever you have time and energy.  Happy to help beta test if you need another pair of eyes 👍

**Comment by jasongaunt** [comment]
@denilsonsa We've now found that the plugin has been hard blocked, please fix this 😊

**Comment by wesleydarnell** [comment]
It's a sad day that chrome has fully broke v2 extensions :(
Miss this auto re-loader already.
@denilsonsa Is a v3 upgrade on the roadmap?

**Comment by thecannons** [comment]
Yup, it's unfortunate v2 extensions are no longer supported. I use this app everyday for work over the last few years.

**Comment by droid192** [comment]
great great extension, simple open source, but it broke now

<img width="1021" height="310" alt="Image" src="https://github.com/user-attachments/assets/127924d9-e711-456d-986a-2a2d3c0da422" />
